samples/client/echo_api/typescript-axios/build/api.ts

```diff
@@ -615,7 +615,7 @@ export const BodyApiAxiosParamCreator = function (configuration?: Configuration)
 
             if (files) {
                 files.forEach((element) => {
-                  localVarFormParams.append('files', element as any);
+                    localVarFormParams.append('files', element as any);
```

Comment:
this looks unrelated

Reply:
Hello,

Yes it looks unrelated.
I had the build failing and I had to run 
the command : 

bash bin/generate-samples.sh 

after this run, this command only added this space change.